frontend/src/plugins/vuetify.js

```diff
@@ -4,6 +4,16 @@ import 'vuetify/dist/vuetify.min.css'
 
 Vue.use(Vuetify)
 
-const opts = {}
+const opts = {
+    theme: {
+        themes: {
+          light: {
+            primary: '#3f51b5',
+            secondary: '#ffe6ff',
+            anchor: '#8c9eff',
```

Comment:
So is this a default vuetify configuration method? Like if we add a `dark: { ... }` would we be able to toggle between those modes?